content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md

```diff
@@ -29,17 +29,18 @@ Using Amazon CloudWatch Metric Streams and Amazon Kinesis Data Firehose, you can
    - Create a CloudWatch Metric Stream linked to your delivery stream.
    - Optionally specify a limited set of namespaces to stream metrics.
 2. Once you create these resources, Datadog immediately starts receiving the streamed metrics and displays them in the Datadog application with no additional configuration needed.
-   - **Note**: Per-namespace defaults and account-level settings in the AWS Integration tile only apply to the API polling approach. Manage all rules for including and excluding namespaces in the streams using the CloudWatch Metric Streams configuration in your AWS accounts.
-   - If you already receive metrics for a given CloudWatch namespace through the API polling method, Datadog automatically detects this and stops polling metrics for that namespace since you are now streaming them. This could lead to some differences in the specific metric names collected as noted in [Supported Metrics](?tab=cloudformation#supported-metrics).
-   - If you later decide you don't want to stream a metric and delete the stream or remove namespaces from it, Datadog automatically starts collecting those metrics using API polling again, according to your configuration settings in the AWS Integration tile.
- 
-### Supported metrics
-Nearly all of the CloudWatch namespaces and metrics that Datadog supports via the API polling approach are also supported via Metric Streams. There are however a couple exceptions.
+   
+   
+### API Polling vs Metric Streaming 
 
-The following CloudWatch metrics are not currently supported:
+   - **Namespace filtering on AWS**: Per-namespace defaults and account-level settings in the AWS Integration tile only apply to the API polling approach. Manage all rules for including and excluding namespaces in the streams using the CloudWatch Metric Streams configuration in your AWS accounts.
+   - **Metrics Collected**: If you already receive metrics for a given CloudWatch namespace through the API polling method, Datadog automatically detects this and stops polling metrics for that namespace since you are now streaming them. This could lead to some differences in the specific metric names collected. For example: __ 
```

Comment:
I think this bullet should be renamed something like "Switching to Metric Streams" and then include the first sentence "If you already receive metrics for a given CloudWatch namespace through the API polling method, Datadog automatically detects this and stops polling metrics for that namespace since you are now streaming them." as well as we should add the importance of not changing any of your existing setting in the tile since we still use polling to collect custom tags and other metadata.

Let's remove "This could lead to some differences in the specific metric names collected. For example: __ " from this section and move it to the warning section